core/sail/fts/lucene-api/src/main/java/org/eclipse/rdf4j/sail/lucene/LuceneSail.java

```diff
@@ -387,6 +388,14 @@ public void initialize()
 		}
 	}
 
+        /**
+         * The method is relocated to {@link SearchIndexUtils#createSearchIndex(java.util.Properties) }.
+         * @param parameters
+         * @return
+         * @throws Exception
+         * @deprecated
+         */
+        @Deprecated
```

Comment:
Is the body of this method able to be replaced with a call to SearchIndexUtils.createSearchIndex to reduce double coding?